deploy/aws/boot.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Check if Lock File exists, if not create it and set trap on exit
+if { set -C; 2>/dev/null >/home/ubuntu/.appsmith.lock; }; then
+    trap "rm -f /home/ubuntu/.appsmith.lock" EXIT
+else
+    exit
+fi
+
+start_docker() {
+    if [ `systemctl is-active docker.service` == "inactive" ];then
+        echo "Starting docker"
+        `systemctl start docker.service`
```

Comment:
Is the backticks here deliberate? We really want to run the output of the `systemctl` command, as a command itself? If not, can we remove the backticks please?

```suggestion
        systemctl start docker.service
```

Reply:
Yes, backticks are deliberate.
Sometimes I face issues in centos/RHEL without backticks. Just take the case in mind I did so.
Even without backticks, it's working fine. I will remove these in the next commit.

Reply:
Interesting. If you come across that issue in CentOS/RHEL again, please share the details. Thanks.